# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/sandbox/linux/reporter/SandboxReporterCommon.h
# Commit: 493b443954fe
# Full Hash: 493b443954fe15f7b542ba14671f25e5f8531dff
# Author: Jed Davis <jld@mozilla.com>
# Date: 2019-02-28 10:55:23
# Regressor Bug: 1506291
# File Overlap Count: 1
# Description:
#   Bug 1506291 - Add Linux sandboxing for the RDD (media decoder) process. r=gcp,mjf,flod
#   
#   The seccomp-bpf policy is currently just the "common" policy with no
#   additions (but with the fixes in bug 1511560 to enable shared memory
#   creation).  The file broker policy allows shared memory creation and
# ==============================================================================

diff -r bf58d8320f5a -r 493b443954fe security/sandbox/linux/reporter/SandboxReporterCommon.h
--- a/security/sandbox/linux/reporter/SandboxReporterCommon.h	Mon Feb 25 16:20:50 2019 +0000
+++ b/security/sandbox/linux/reporter/SandboxReporterCommon.h	Wed Feb 27 20:14:54 2019 +0000
@@ -33,6 +33,7 @@
     CONTENT,
     FILE,
     MEDIA_PLUGIN,
+    RDD,
   };
 
   // The syscall number and arguments are usually `unsigned long`, but